setup.py

```diff
@@ -228,6 +228,14 @@ def run(self):
         print("Unsupported command for openshift-ansible")
 
 
+def extract_requirements(filename):
+    """
+    Extracts requirements from a pip formatted requirements file.
+    """
+    with open(filename, 'r') as requirements_file:
+        return [x.split()[0] for x in requirements_file.readlines()]
```

Comment:
It's fine to drop with no one else needs the workflow. It tripped me up when I joined hence why I added it.